selfdrive/ui/qt/maps/map.cc

```diff
@@ -362,6 +364,7 @@ void MapWindow::offroadTransition(bool offroad) {
 }
 
 void MapWindow::updateDestinationMarker() {
+  m_settings.setApiKey(get_mapbox_token());
```

Comment:
you sure this updates the settings for the map widget?

Reply:
yes, The `MapWindow` class has both `m_settings` and `m_map` as private and since we're updating the setting within the class it would be updating the setting of the MapLibre widget
```
class MapWindow : public QOpenGLWidget {
  Q_OBJECT

public:
  MapWindow(const QMapLibre::Settings &);
  ~MapWindow();

private:
  void initializeGL() final;
  void paintGL() final;
  void resizeGL(int w, int h) override;

  QMapLibre::Settings m_settings;
  QScopedPointer<QMapLibre::Map> m_map;
```

An example of this can be seen with `m_map` where we make updates to the `QMapLibre::map` itself such as setting the zoom and coordinates:
```
  if (interaction_counter == 0) {
    if (last_position) m_map->setCoordinate(*last_position);
    if (last_bearing) m_map->setBearing(*last_bearing);
    m_map->setZoom(util::map_val<float>(velocity_filter.x(), 0, 30, MAX_ZOOM, MIN_ZOOM));
  } else {
    interaction_counter--;
  }
```